Cargo.toml

```diff
@@ -17,4 +17,4 @@ default = ["use_std"]
 use_std = []
 
 [dev-dependencies]
-quickcheck = "^0.5.0"
+quickcheck = "^1.0.0"
```

Comment:
Can this also be written as `"1"` ?

Reply:
> I think this can also be written as `"1"` , which is a bit simpler

Indeed, those should be the same (https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#version-requirement-syntax). I’m happy to adjust the style.

Reply:
Done! Thanks for the review.